config/samples/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+## Append samples you want in your CSV to this file as resources ##
```

Comment:
is the CSV also being generated? and if so, will this be for the alm-examples? because for certified we use different alm-examples

Reply:
we don't generate the csv/cr yaml files I will remove it

Reply:
shouldn't we generate them as well?

Reply:
opened a ticket for it
CSI-3250

Reply:
I don't see anything regarding the cr there

Reply:
I removed this file

Reply:
I meant that I don't see anything regarding the cr in the ticket

Reply:
added reference to the CR